backend/src/controllers/upload.ts

```diff
@@ -5,6 +5,7 @@ import multer from "multer";
 import { FolderTreeFolderSchema } from "../models/objects/folderTree.js";
 import { uploadOptionsSchema } from "../models/uploads/upload.js";
 import { z } from "zod";
+import { cidToString } from "@autonomys/auto-drive";
```

Comment:
This line is useless
Furthermore, can you add workflow to run eslint to catch these things as part of our ci/cd ?